forge/test/mlir/llama/tests/test_llama_embedding.py

```diff
@@ -5,14 +5,18 @@
 import pytest
 
 import forge
-from test.mlir.llama.utils.utils import load_model
+from test.mlir.llama.utils.utils import load_model, load_llama32
 from forge.op.eval.common import compare_with_golden_pcc
 
 
-@pytest.mark.xfail()
-def test_llama_embedding():
-    # Load Llama 3B model and tokenizer
-    framework_model, _ = load_model()
+@pytest.mark.parametrize("llama_ver", ["llama 3B", "llama 3.2 1B"])
+@pytest.mark.xfail(reason="Waiting for the transformers version to be upgraded")
```

Comment:
This is probably the case for Llama 3.2 1B only. Let's detect that in test itself, and xfail with this msg only for that variant. 

Let's leave Open Llama 3B unblocked for further push :))

Same comment applies for other Llama module tests

Reply:
fixed